src/index.html

```diff
@@ -2,11 +2,506 @@
+        <div class="catalog__cards">
+
+          <article
+            class="catalog__card product-card"
+            data-qa="card">
+            <img
+              src="images/imac.jpeg"
+              alt="iMac"
+              class="product-card__img"
+            >
+
+            <div class="product-card__data">
+              <h2 class="product-card__title">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </h2>
+              <p class="product-card__product-number">
+                Код товара: 195434
+              </p>
+            </div>
+
+            <div class="product-card__ratings">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <span class="product-card__reviews-count">
+                Отзывов: 5
+              </span>
+            </div>
+
+            <div class="product-card__price-info">
+              <span class="product-card__price-caption">
+                Цена:
+              </span>
+              <span class="product-card__price">
+                69 999 грн
+              </span>
+            </div>
+
+            <a
+              href="#"
+              class="button"
+              data-qa="card-hover"
+            >
+            Купить
+            </a>
+          </article>
+
+          <article class="catalog__card product-card" >
+            <img
+              src="images/imac.jpeg"
+              alt="iMac"
+              class="product-card__img"
+            >
+
+            <div class="product-card__data">
+              <h2 class="product-card__title">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </h2>
+              <p class="product-card__product-number">
+                Код товара: 195434
+              </p>
+            </div>
+
+            <div class="product-card__ratings">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <span class="product-card__reviews-count">
+                Отзывов: 5
+              </span>
+            </div>
+
+            <div class="product-card__price-info">
+              <span class="product-card__price-caption">
+                Цена:
+              </span>
+              <span class="product-card__price">
+                69 999 грн
+              </span>
+            </div>
+
+            <a
+              href="#"
+              class="button"
+              data-qa="hover"
+            >
+            Купить
```

Comment:
```suggestion
              Купить
```
don't have 2 whitespace